assets/generation/pip/requirements.txt

```diff
@@ -8,8 +8,8 @@
 # Update me and then run 'docker-compose -f dcp-generation.yml run assets pip_freeze'	#
 #                                                                                       #
 #########################################################################################
-aether.client==0.0.0
-aether.mocker==0.0.0
+aether.client==0.10.0
+aether.mocker==0.10.0
```

Comment:
I thought you were going to remove the `aether.mocker` module :thinking:

Reply:
The asset generator uses `aether.mocker`, which is still good at making arbitrary amounts of fake entities whose fields can be set by a specific generator function. Like `.lat` is a random value from a gaussian distribution centered at this point in space. We don't use `aether.mocker` in integration tests anymore in the interest of speed.

Reply:
My question in that case is, could you move [mocker.py](https://github.com/eHealthAfrica/aether/blob/develop/aether-utils/aether-mock-data/aether/mocker.py) file here? 
If it's the only place in which is used and we can remove the package in `aether`.

Does it make sense for you? Will it simplify this `generation` container? :thinking:

Reply:
@obdulia-losantos; as usual you're right. We should also make a todo for adding travis to this repo so we can run generation and do some quick checks so that don't release anything broken.

Reply:
Actually, Travis is already there :https://github.com/eHealthAfrica/aether-bootstrap/blob/master/.travis.yml

:blush:

Reply:
Oh nice. I can just add a small integration check then.

Reply:
@obdulia-losantos Mocker moved to this repo. Will PR it out of aether-develop